requirements.txt

```diff
@@ -3,8 +3,10 @@ django-debug-toolbar==3.4.0
 djangorestframework==3.13.1
 djangorestframework-simplejwt==5.2.0
 drf-spectacular==0.22.1
-Pillow==9.1.1
+Pillow
```

Comment:
It is better to define specific versions of libraries than rely on the latest ones to ensure the project's workability.